lib/ocpp1_6/database_handler.cpp

```diff
@@ -0,0 +1,601 @@
+
+#include <everest/logging.hpp>
+
+#include "ocpp1_6/database_handler.hpp"
+
+namespace ocpp1_6 {
+
+DatabaseHandler::DatabaseHandler(const std::string& chargepoint_id, const boost::filesystem::path& database_path,
+                                 const boost::filesystem::path& init_script_path) {
+    const auto sqlite_db_filename = chargepoint_id + ".db";
+    if (!boost::filesystem::exists(database_path)) {
+        boost::filesystem::create_directories(database_path);
+    }
+    this->db_path = database_path / sqlite_db_filename;
+    this->init_script_path = init_script_path;
+}
+
+DatabaseHandler::~DatabaseHandler() {
+    if (sqlite3_close_v2(this->db) == SQLITE_OK) {
+        EVLOG_info << "Successfully closed database file";
+    } else {
+        EVLOG_error << "Error closing database file: " << sqlite3_errmsg(this->db);
+    }
+}
+
+void DatabaseHandler::open_db_connection(int32_t number_of_connectors) {
+    if (sqlite3_open(this->db_path.c_str(), &this->db) != SQLITE_OK) {
+        EVLOG_error << "Error opening database: " << sqlite3_errmsg(db);
+        throw std::runtime_error("Could not open database at provided path.");
+    }
+    EVLOG_info << "Established connection to Database.";
```

Comment:
Does this (and similar log messages in this file) really have to be of loglevel INFO?